contracts/Election.sol

```diff
@@ -96,24 +96,24 @@ contract Election {
         address highestCandidate;
         for (uint x = 0; x < numOfCandidates; x++) {
             address currentAddr = candiatesList[x];
-            Candidate storage currentCandidate = candidateData[round][currentAddr];
+            Candidate memory currentCandidate = candidateData[round][currentAddr];
```

Comment:
`memory` here will cause more gas fee. when you assign `storage` data to  `memory`, it basically copy data from storage and then allocate new memory location to store the data

Reply:
oh!!!! got it! thanks, let me change it back!